salt/proxy/junos.py

```diff
@@ -122,27 +122,19 @@ def proxytype():
     '''
     return 'junos'
 
-
-def grains():
```

Comment:
No, it won't. The grains function was used in salt/grains/junos.py to set the junos_facts grains. Now instead of grains(), get_serialized_facts() would be used. See this:
https://github.com/rajvidhimar/salt/blob/bdc8d2d196417a4ce9247d5ea9a9770ab5d1adae/salt/grains/junos.py#L51